app/src/main/res/layout/add_edit_page.xml

```diff
@@ -1,25 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fillViewport="true">
+
```

Comment:
make edit mood a scroll view to resolve parts not showing on small phone problem